ftw/tika/converter.py

```diff
@@ -100,11 +100,19 @@ def convert(self, document, filename=''):
         if self.server_configured:
             try:
                 text = self.convert_server(document, filename)
-            except (socket.error, RequestException), exc:
+
+            except requests.exceptions.ConnectionError:
+                # Could not connect to the server at all.
+                # Server is probably not running
                 self.log.error(
                     'Could not connect to tika server: %s' % str(exc))
-                # Use local tika as fallback
                 text = self.convert_local(document, filename)
+
+            except (socket.error, RequestException), exc:
+                # An error occured, maybe a connection timeout.
+                self.log.error(
+                    'Connection error: %s' % str(exc))
```

Comment:
This is not a connection error (any more), that's being caught just above. Suggestion: `Full text extraction using tika failed: %s`. Also makes it more clear that searchable text is now empty.

Reply:
Thanks, updated 👍